<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Music Machine 2.2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pauseAllButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/control_pause_blue.png</normaloff>:/icons/control_pause_blue.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopAllButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/control_stop_blue.png</normaloff>:/icons/control_stop_blue.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>L1 00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LayerSelector" name="layerSelector" native="true">
            <property name="styleSheet">
             <string notr="true">border:1px solid black;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="SlotTableWidget" name="slotTableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackedWidgetPage2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QWidget" name="playlistToolsWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="openButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QColumnView" name="fileView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragOnly</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
          <widget class="QWidget" name="playlistWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="PlaylistWidget" name="playListTable">
               <property name="styleSheet">
                <string notr="true">font-size:20px;</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>40</number>
               </attribute>
              </widget>
              <widget class="QWidget" name="historyWidget" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>History:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="HistoryListWidget" name="historyList"/>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="saveHistoryButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="playlistControlWidget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="autoPlayCheckBox">
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/arrow_switch.png</normaloff>:/icons/arrow_switch.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="playlistNextButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font-size:12px;
font-weight:bold;</string>
                 </property>
                 <property name="text">
                  <string>NEXT</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/document_page_next.png</normaloff>:/icons/document_page_next.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="playerWidget" native="true">
              <layout class="QHBoxLayout" name="playerLayout">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="configAction"/>
    <addaction name="slotStoreAction"/>
    <addaction name="setConfigurationAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string notr="true">?</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuDiverses">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="colorAction"/>
    <addaction name="clearLayerAction"/>
    <addaction name="resetCountersAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuDiverses"/>
   <addaction name="menu"/>
  </widget>
  <action name="configAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/setting_tools.png</normaloff>:/icons/setting_tools.png</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="editAction">
   <property name="text">
    <string>Slots editieren</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>Über die Event Music Machine</string>
   </property>
  </action>
  <action name="slotAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/small_tiles.png</normaloff>:/icons/small_tiles.png</iconset>
   </property>
   <property name="text">
    <string>Slots</string>
   </property>
  </action>
  <action name="playlistAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/control_play_blue.png</normaloff>:/icons/control_play_blue.png</iconset>
   </property>
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="slotStoreAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/server_components.png</normaloff>:/icons/server_components.png</iconset>
   </property>
   <property name="text">
    <string>Slot-Speicher</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>Farben dieses Layers auf alle anderen übertragen</string>
   </property>
  </action>
  <action name="clearLayerAction">
   <property name="text">
    <string>Alle Slots in diesem Layer löschen</string>
   </property>
  </action>
  <action name="setConfigurationAction">
   <property name="text">
    <string>Konfiguration wechseln</string>
   </property>
  </action>
  <action name="resetCountersAction">
   <property name="text">
    <string>Alle Slot-Zähler in diesem Layer zurücksetzen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlaylistWidget</class>
   <extends>QTableWidget</extends>
   <header>view/playlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryListWidget</class>
   <extends>QListWidget</extends>
   <header>view/historylistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LayerSelector</class>
   <extends>QWidget</extends>
   <header>view/layerselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlotTableWidget</class>
   <extends>QTableWidget</extends>
   <header>view/slottablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
